### PR TITLE
Adding organizations to the admin dashboard

### LINKED ISSUE #18: Manage and display organizations
As a company, I'd like to have a page on the website
    Name
    Address
    Logo
    Description
    List of members working there
    List of members who can manage the company

### PR DESCRIPTION
**Pull request by syed**
Adding organizations to the admin dashboard Fixes #18

### CODE REVIEW

**Comment by nicholasjhenry** [comment]
@syed Would love to merge. Are you up for rebasing with master?

**Comment by syed** [comment]
rebase done!

File: db/schema.rb [hunk 1]
```diff
@@ -80,6 +69,7 @@
     t.datetime "updated_at"
   end
 
+<<<<<<< HEAD
```
Comment (nicholasjhenry) [inline db/schema.rb]: @syed Whoops, looks like a conflict got committed by mistake.

Comment (syed) [inline db/schema.rb]: ooops sorry ... Fixed it.

Comment (nicholasjhenry) [inline db/schema.rb]: :+1:

**Comment by nicholasjhenry** [comment]
I believe the CI is failing due to some broken migrations that are now fixed in master -- I've tested this locally on my machine so I believe it's OK to merge.

**Pull request merged**